### PR TITLE
TFL Bus Status Edge App

### PR DESCRIPTION
**Pull request by salmanfarisvp**
The Edge app will showcase the bus arrival time and status at a bus stop. 

image

Features 

- [x] Based on [TFL API](https://api.tfl.gov.uk/) and Bus ID - Fetch **Bus station name**, **Towwards name**, **Platform name** and **Staus** . 
- [x] Bus Platform Number and Bus towards details.  
- [x] List five of the latest bus information for landscape screen orientation and seven for portrait mode. 
- [x] Updates font size depending on the screen size - followed Edge app playground [Resolution guide](https://github.com/Screenly/Playground/blob/master/docs/resolutions.md)
-  [x] Implement Screenly `ready_signal` feature
- [x] Implement API Cache 
- [x] Tracks/captures errors using [Sentry](https://www.sentry.io/)
- [x]  Need to add Edge App Icon

ToDo 
- Nothing for now.

### CODE REVIEW

File: edge-apps/tfl-bus-status/screenly.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+author: Screenly, Inc.
+entrypoint: index.html
+settings:
+  Stop_ID:
+    type: string
+    default_value: 490006428B
+    title: Bus Stop ID
+    optional: false
+    help_text: TFL Bus Stop ID.
+  TFL_API:
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/screenly.yml:14]: The capitalization here does not align with capitalization for other Edge Apps. Let's make it consistent.

File: edge-apps/tfl-bus-status/static/js/script.js [hunk 2]
```diff
@@ -0,0 +1,300 @@
+/* global screenly */
+
+const apiUrl = 'https://api.tfl.gov.uk/' // Base URL for the TfL API
+const stopId = screenly.settings.Stop_ID
+const apiKey = screenly.settings.TFL_API
+
+// for debug local server
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/static/js/script.js:7]: Remove debug comments before final draft. You also leaked an API key below.

File: edge-apps/tfl-bus-status/screenly.yml (hunk 1 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/screenly.yml:14]: Noted. Thanks

File: edge-apps/tfl-bus-status/static/js/script.js (hunk 2 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/static/js/script.js:7]: Oops, my bad. I will revoke the API now.

File: edge-apps/tfl-bus-status/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+> Wait for the upload to complete.
+
+Step 6. **Specify the TFL API and STOP ID** for example: 490005186S1.
+
+Replace "API" with the actual API that obtained from [https://api-portal.tfl.gov.uk/](https://api-portal.tfl.gov.uk/)
+
+`$ screenly edge-app secret set tfl_api=API`
+
+and provide the STOP ID also - replace the "stopID" with actual Stop ID.
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/README.md:44]: Where does one get the stop ID?

File: edge-apps/tfl-bus-status/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Step 3. **Enter tfl-bus-status Edge App Directory**
+
+`$ cd edge-apps/tfl-bus-status`
+
+Step 4. **Create a New tfl-bus-status Edge App:**
+
+`$ screenly edge-app create --name TFL_EdgeApp --in-place`
+
+> Replace "TFL_EdgeApp" with your desired app name.
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/README.md:30]: `>` implies that it's a code block.

File: edge-apps/tfl-bus-status/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+* Install Screenly CLI - Please follow the guide [here](https://github.com/Screenly/cli)
+
+## Steps
+
+Step 1. **Log in to the Screenly account via CLI**
+
+Follow the on-screen instructions to log in to your Screenly account.
+
+   `$ screenly login`
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/README.md:16]: Either do four indentations (means code), use the tick marks (single command) or use a code block. This is redundant.

**Review by vpetersson — APPROVED** [review]
Approved pending minor comments.

File: edge-apps/tfl-bus-status/README.md (hunk 4 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/README.md:30]: Removing > as the content it's not part of code block.

File: edge-apps/tfl-bus-status/README.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+`$ git clone https://github.com/Screenly/Playground.git`
+
+Step 3. **Enter tfl-bus-status Edge App Directory**
+
+`$ cd edge-apps/tfl-bus-status`
+
+Step 4. **Create a New tfl-bus-status Edge App:**
+
+`$ screenly edge-app create --name TFL_EdgeApp --in-place`
```
Comment (vpetersson) [inline edge-apps/tfl-bus-status/README.md:28]: Nit but `$ screenly edge-app create --name "TFL Edge App" --in-place` probably looks better.

Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/README.md:28]: Thanks. Updating.

File: edge-apps/tfl-bus-status/README.md (hunk 3 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/README.md:44]: Currently, there is an[ official CSV file](https://tfl.gov.uk/bus-stops.csv ) provided by the TFL team. But it's not user-friendly as the user needs to download the .csv file and needs to search for the stop name or stopID. I'm looking for a good alternative, or we might need to make a static page.

Comment (vpetersson) [reply edge-apps/tfl-bus-status/README.md:44]: Sorted in [here](https://playground.srly.io/edge-apps/helpers/tfl/bus-stop-lookup/) for now.

**Review by vpetersson — APPROVED** [review]

File: edge-apps/tfl-bus-status/README.md (hunk 3 above)
Comment (salmanfarisvp) [reply edge-apps/tfl-bus-status/README.md:44]: Thanks @vpetersson , Will improve the UI later.

**Pull request merged**